src/App.css

```diff
@@ -15,13 +14,86 @@ a {
   color: inherit;
 }
 
```

Comment:
- [ ] [REQUIRED] Good job so far! As previous reviewer mentioned about your main box of project should be give `shadow` property and pay attention to `distance of elements from the border of box, in your project I see these aspects are not fixed yet. Please improve them 🙏🏼 

![image](https://github.com/eamador1/bookstore-redux/assets/63915024/03e568a4-4d5a-4beb-abea-3d6a53e5a37c)
  
![image](https://github.com/eamador1/bookstore-redux/assets/63915024/12fecb0d-f5d5-4318-8cfe-de5c97260899)